### DISCUSSION TITLE
Incorrect behaviour with Activate<tooltype>ToolWindow after upgrade to IntelliJ 2023.2

### DISCUSSION THREAD

**Discussion by rbhanot4739** [JetBrains/ideavim#688] [Q&A]
Hello,

Here is the relevant section from `ideavimrc`  which maps some actions for activating the tool windows

```
map <C-A-p> :<Action>(ActivateProjectToolWindow)
nnoremap <C-A-s> :action ActivateStructureToolWindow<CR>
nnoremap <C-A-t> :action ActivateTerminalToolWindow<CR>
``` 
With IJ 2022.2 when I press any of the above key sequences it would activate the respective tool window and then pressing it again would hide it i.e. it basically behaved as a `toogle` action.

However I recently installed IJ 2023.2 and the toggle behaviour no longer works, so pressing the key sequence would open the tool window if its not already open and pressing it again does nothing.

Here are the two gifs for both the versions

**IJ 2023.2**

![2023 2](https://github.com/JetBrains/ideavim/assets/19793591/0bd3e05a-6ba4-40be-85a7-ed32468f484c)

**IJ 2022.2**

![2022 2](https://github.com/JetBrains/ideavim/assets/19793591/e1922830-2bbf-4446-8742-6cef5862c65a)

**Comment by rbhanot4739** [comment]
Is there anyone else also seeing this behaviour ?

**Reply by rbhanot4739** [reply]
Can someone please respond on this ?

**Reply by AlexPl292** [reply]
This seems to be a change in the IntelliJ platform. You can ask about that here: https://youtrack.jetbrains.com/issues/IDEA
